### ISSUE TITLE
Tooltip  data-tip value is converted to upper case when used in Navbar 

### ISSUE THREAD

**Issue by rhscjohn-dev** [saadeghi/daisyui#304]
The value of data-tip, when used in  the NavBar component,   is converted to uppercase.  See attached screen shot.

Also notice that the tooltip(tooltip-bottom utility), applied to a button located on the far right of the screen overflows to the right and is clipped.   There are also issues when using tooltip-top and tooltip-right utility class names.

There appears to be no reduction is font size.

Here is the Svelte component.
```javascript
<script>
  import { onMount } from "svelte";

  import "../app.css";
  let theme = "dark";
  let themes = [
    "light",
    "dark",
    "cupcake",
    "bumblebee",
    "emerald",
    "corporate",
    "valentine",
    "halloween",
    "garden",
    "forest",
    "aqua",
    "dark",
  ];
  let htmlTag;

  function onClickChangeTheme(event) {
    console.log("onClickChangeTheme");
    htmlTag.setAttribute("data-theme", theme);
  }
  onMount(() => {
    console.log("onMount:__layout.svelte");
    htmlTag = document.getElementsByTagName("html")[0];
    theme = htmlTag.getAttribute("data-theme");
    console.log("onMount:__layout.svelte data-theme: ", theme);
  });
</script>

<div class="navbar mb-2 shadow-lg bg-neutral text-neutral-content rounded-box ">
  <!-- navbar start -->
  <div class="px-2 mx-2 navbar-start">
    <div class="flex-none w-8 h-8">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 103 124">
        <path
          style="fill:#ff3e00"
          d="M96.33,20.61C85.38,4.93,63.74.28,48.09,10.25L20.61,27.77A31.46,31.46,0,0,0,6.37,48.88,33.22,33.22,0,0,0,9.64,70.2,31.52,31.52,0,0,0,4.93,82a33.61,33.61,0,0,0,5.73,25.41c11,15.68,32.6,20.33,48.25,10.36l27.48-17.52a31.48,31.48,0,0,0,14.24-21.11A33.22,33.22,0,0,0,97.36,57.8,31.52,31.52,0,0,0,102.07,46a33.57,33.57,0,0,0-5.74-25.41"
        />
        <path
          style="fill:white"
          d="M45.41,108.86A21.81,21.81,0,0,1,22,100.18,20.2,20.2,0,0,1,18.53,84.9a19,19,0,0,1,.65-2.57l.52-1.58,1.41,1a35.32,35.32,0,0,0,10.75,5.37l1,.31-.1,1a6.2,6.2,0,0,0,1.11,4.08A6.57,6.57,0,0,0,41,95.19a6,6,0,0,0,1.68-.74L70.11,76.94a5.76,5.76,0,0,0,2.59-3.83,6.09,6.09,0,0,0-1-4.6,6.58,6.58,0,0,0-7.06-2.62,6.21,6.21,0,0,0-1.69.74L52.43,73.31a19.88,19.88,0,0,1-5.58,2.45,21.82,21.82,0,0,1-23.43-8.68A20.2,20.2,0,0,1,20,51.8a19,19,0,0,1,8.56-12.7L56,21.59a19.88,19.88,0,0,1,5.58-2.45A21.81,21.81,0,0,1,85,27.82,20.2,20.2,0,0,1,88.47,43.1a19,19,0,0,1-.65,2.57l-.52,1.58-1.41-1a35.32,35.32,0,0,0-10.75-5.37l-1-.31.1-1a6.2,6.2,0,0,0-1.11-4.08,6.57,6.57,0,0,0-7.06-2.62,6,6,0,0,0-1.68.74L36.89,51.06a5.71,5.71,0,0,0-2.58,3.83,6,6,0,0,0,1,4.6,6.58,6.58,0,0,0,7.06,2.62,6.21,6.21,0,0,0,1.69-.74l10.48-6.68a19.88,19.88,0,0,1,5.58-2.45,21.82,21.82,0,0,1,23.43,8.68A20.2,20.2,0,0,1,87,76.2a19,19,0,0,1-8.56,12.7L51,106.41a19.88,19.88,0,0,1-5.58,2.45"
        />
      </svg>
    </div>
    <div class="flex-1 px-2 mx-2">
      <span class="text-lg font-bold"> SvelteKit with Daisy UI </span>
    </div>
  </div>
  <!-- navbar center -->
  <div class="hidden px-2 mx-2 navbar-center md:flex">
    <div class="flex items-stretch  ">
      <a href="/" class="btn btn-ghost btn-sm rounded-btn mt-2  "> Home </a>
      <a href="/alerts" class="btn btn-ghost btn-sm rounded-btn mt-2">
        Alerts
      </a>
      <a href="/buttons" class="btn btn-ghost btn-sm rounded-btn mt-2">
        Buttons
      </a>
      <a href="/cards" class="btn btn-ghost btn-sm rounded-btn mt-2"> Cards </a>
      <a href="/colors" class="btn btn-ghost btn-sm rounded-btn mt-2">
        Colors
      </a>
      <a href="/modal" class="btn btn-ghost btn-sm rounded-btn mt-2"> Modal </a>
      <a href="/test" class="btn btn-ghost btn-sm rounded-btn mt-2"> Test </a>
      <div class="dropdown dropdown-end mt-0">
        <div tabindex="0" class="btn btn-ghost rounded-btn">Input</div>
        <ul
          tabindex="0"
          class="p-2 shadow menu dropdown-content bg-base-100 text-base-content rounded-box w-25"
        >
          <li>
            <a href="/input/text">Text</a>
          </li>
          <li>
            <a href="/input/toggle">Toggle</a>
          </li>
          <li>
            <a href="/input/radio">Radio</a>
          </li>
          <li>
            <a href="/input/checkbox">Checkbox</a>
          </li>
          <li>
            <a href="/input/select">Select</a>
          </li>
        </ul>
      </div>
    </div>
  </div>
  <!-- navbar end -->
  <div class="navbar-end ">
    <button
      on:click={onClickChangeTheme}
      data-tip="theme: {theme}"
      class="btn btn-square btn-ghost tooltip tooltip-bottom "
    >
      <svg
        class="fill-current text-neutral-content"
        xmlns="http://www.w3.org/2000/svg"
        height="24"
        width="24"
        ><path fill="none" d="M0 0h24v24H0z" /><path
          d="M9.37 5.51A7.35 7.35 0 0 0 9.1 7.5c0 4.08 3.32 7.4 7.4 7.4.68 0 1.35-.09 1.99-.27A7.014 7.014 0 0 1 12 19c-3.86 0-7-3.14-7-7 0-2.93 1.81-5.45 4.37-6.49zM12 3a9 9 0 1 0 9 9c0-.46-.04-.92-.1-1.36a5.389 5.389 0 0 1-4.4 2.26 5.403 5.403 0 0 1-3.14-9.8c-.44-.06-.9-.1-1.36-.1z"
        /></svg
      >
    </button>
  </div>
</div>
<slot />
```

Maybe add a new utility class called tooltip-font-size which would allow overriding the inherited font size.

![Screenshot 2021-11-15 164106](https://user-images.githubusercontent.com/60523782/141860061-f1ef32b1-7af3-4e6c-a3bc-c66747ba45cc.png)

**Comment by saadeghi** [comment]
@rhscjohn-dev Well, it's not just font size...  
When the `btn` and the `tooltip` wrapper are the same element, everything inside will have the style of a button text (size, font, weight, etc...) and adding  new utility classes to reset all those styles is not efficient and flexible.  
The correct thing here is to use a `<span class="tooltip" data-tip="text">` and then put a <button class="btn"> inside that.

Please let me know if you have any questions.

**Comment by saadeghi** [comment]
This is fixed in version `2.0`
